### PR TITLE
Increased speed ot writing the json logs

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

Lots of ignores because of a mypy bug with lru_cache function

### CODE REVIEW

File: demisto_sdk/commands/common/tools.py [hunk 1]
```diff
@@ -965,6 +965,7 @@ def get_dict_from_file(path: str, use_ryaml: bool = False) -> Tuple[Dict, Union[
 
 
 # flake8: noqa: C901
+@lru_cache()
```
Comment (roysagi) [inline demisto_sdk/commands/common/tools.py:968]: what happens when I run on several files?

File: demisto_sdk/commands/common/tools.py [hunk 2]
```diff
@@ -1638,6 +1639,7 @@ def find_file(root_path, file_name):
     return ''
 
 
+@lru_cache()
```
Comment (roysagi) [inline demisto_sdk/commands/common/tools.py:1642]: same question as above

File: demisto_sdk/commands/lint/lint_manager.py [hunk 3]
```diff
@@ -739,6 +740,7 @@ def create_json_output(self):
         logger.info('Collecting results to write to file')
         # format all linters to JSON format -
         # if any additional linters are added, please add a formatting function here
+        start = timeit.default_timer()
```
Comment (roysagi) [inline demisto_sdk/commands/lint/lint_manager.py:743]: seems redundant no?

File: demisto_sdk/commands/common/tools.py (hunk 1 above)
Comment (jochman) [reply demisto_sdk/commands/common/tools.py:968]: it'll cache the result in the memory. if it's for different files, it'll store a map of input -> output. 
Least Recently Used - Will delete things that are not in use.

File: demisto_sdk/commands/lint/lint_manager.py (hunk 3 above)
Comment (jochman) [reply demisto_sdk/commands/lint/lint_manager.py:743]: seems like a leftover

**Review by roysagi — APPROVED** [review]

**Pull request merged**